### PR TITLE
ResNet50 || ImageNet Inference || Model Arts Service of Huawei Cloud

### PR DESCRIPTION
**Pull request by terminatorzwm**
Huawei Cloud submissions for ImageNet inference with resnet50 on ecs.pi1.2xlarge.4

### CODE REVIEW

File: ImageNet/inference/HuaweiCloud_resnet50_modelarts_2.4.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "version": "v1.0",
+    "author": "ModelArts Service of Huawei Cloud",
+    "authorEmail": "william.zhouweimin@huawei.com",
+    "framework": "TensorFlow 1.8.0",
+    "codeURL": "https://github.com/huawei-clouds/modelarts-example/DAWNBench_Inference",
```
Comment (codyaustun) [inline ImageNet/inference/HuaweiCloud_resnet50_modelarts_2.4.json]: This URL gives a 404 Error

**Review by codyaustun — CHANGES_REQUESTED** [review]
The `codeURL` is incorrect. Also, make sure to include the commit hash.

File: ImageNet/inference/HuaweiCloud_resnet50_modelarts_2.4.json (hunk 1 above)
Comment (terminatorzwm) [reply ImageNet/inference/HuaweiCloud_resnet50_modelarts_2.4.json]: Hi, thanks for your efforts on this PR.
The right code URL and commit hash have been updated.

File: ImageNet/inference/HuaweiCloud_resnet50_modelarts_2.4.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "v1.0",
+    "author": "ModelArts Service of Huawei Cloud",
+    "authorEmail": "william.zhouweimin@huawei.com",
+    "framework": "TensorFlow 1.8.0",
+    "codeURL": "https://github.com/huawei-clouds/modelarts-example/tree/master/DAWNBench_Inference",
+    "commitHash": "https://github.com/huawei-clouds/modelarts-example/commit/b8831d3e3e38ccbdbed7b7e8bc697622e02dc069",
```
Comment (codyaustun) [inline ImageNet/inference/HuaweiCloud_resnet50_modelarts_2.4.json]: Remove the rest of the URL

```suggestion
    "commitHash": "b8831d3e3e38ccbdbed7b7e8bc697622e02dc069",
```

**Review by codyaustun — CHANGES_REQUESTED** [review]

File: ImageNet/inference/HuaweiCloud_resnet50_modelarts_2.4.json (hunk 2 above)
Comment (terminatorzwm) [reply ImageNet/inference/HuaweiCloud_resnet50_modelarts_2.4.json]: Hi, commitHash has been updated. Please check again, thanks.

**Review by codyaustun — APPROVED** [review]

**Pull request merged**